docs/integrating/working-with-libraries.md

```diff
@@ -0,0 +1,59 @@
+# Working with libraries
+This document is general advice for creating new components and utilities.
+
+## Table of Contents
+
+- [Community definitions](#community-definitions)
+- [Declaring definitions](#ceclaring-definitions)
```

Comment:
`ceclaring` => `declaring`

Reply:
Wait, most of these ids don't match the heading text, might want to double check them.